qiita_core/support_files/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Have these been tested when serving from other systems? I want to make sure they don't trigger HTTPS error pages.